bioblend/galaxy/folders/__init__.py

```diff
@@ -56,11 +83,66 @@ def show_folder(self, folder_id: str, contents: bool = False) -> Dict[str, Any]:
         :param contents: True to get the contents of the folder, rather
           than just the folder details.
 
+        :type limit: int
+        :param limit: When ``contents=True``, maximum number of items to return.
+
+        :type offset: int
+        :param contents: When ``contents=True``, number of items to skip. Return
+          contents starting with item offset+1.
```

Comment:
I'm trying to improve consistency in https://github.com/nsoranzo/bioblend/commit/b58b0a3612d6c3c50953faa7834303fd3f41a3e7 and I think `starting from` is more correct English, sorry for the back and forth!

```suggestion
          contents starting from item offset+1.
```